kolibri/plugins/coach/assets/src/modules/exerciseDetail/actions.js

```diff
@@ -1,14 +1,15 @@
 import { AttemptLogResource } from 'kolibri.resources';
 import { assessmentMetaDataState } from 'kolibri.coreVue.vuex.mappers';
 
-export function setAttemptLogs(store, params) {
-  const { learnerId, exercise } = params;
+export function setAttemptLogs(store) {
+  const { learnerId, exercise, attemptId } = store.state;
   return AttemptLogResource.fetchCollection({
     getParams: {
       user: learnerId,
       content: exercise.content_id,
     },
-    force: true,
+    // Only force when accessing the root url
+    force: attemptId === null,
```

Comment:
would this depend in part on where they're coming _from_? e.g. would a direct link to a non-root URL from outside kolibri not work?

Reply:
It would still work, it just wouldn't force the fetch. If someone refreshed the page, or pasted a link, it would still do the fetch.